static/docker-compose.yml

```diff
@@ -48,24 +49,8 @@ services:
          npm run db:migrate &&
          npm run pg:etl up all &&
          npm run maintenance:disable"
-    environment:
-      COUCHBASE_URL: ${COUCHBASE_URL:-http://plextracdb}
-      CB_API_PASS: ${CB_API_PASS}
-      CB_API_USER: ${CB_API_USER}
-      REDIS_CONNECTION_STRING: ${REDIS_CONNECTION_STRING:-redis}
-      REDIS_PASSWORD: ${REDIS_PASSWORD:?err}
-      PG_HOST: ${PG_HOST:-postgres}
-      PG_MIGRATE_PATH: /usr/src/plextrac-api
-      PG_CORE_ADMIN_PASSWORD: ${PG_CORE_ADMIN_PASSWORD:?err}
-      PG_CORE_ADMIN_USER: ${PG_CORE_ADMIN_USER:?err}
-      PG_CORE_DB: ${PG_CORE_DB:?err}
-      PG_RUNBOOKS_ADMIN_PASSWORD: ${PG_RUNBOOKS_ADMIN_PASSWORD:?err}
-      PG_RUNBOOKS_ADMIN_USER: ${PG_RUNBOOKS_ADMIN_USER:?err}
-      PG_RUNBOOKS_RW_PASSWORD: ${PG_RUNBOOKS_RW_PASSWORD:?err}
-      PG_RUNBOOKS_RW_USER: ${PG_RUNBOOKS_RW_USER:?err}
-      PG_RUNBOOKS_DB: ${PG_RUNBOOKS_DB:?err}
-      PG_TENANTS_WRITE_MODE: ${PG_TENANTS_WRITE_MODE:-couchbase_only}
-      PG_TENANTS_READ_MODE: ${PG_TENANTS_READ_MODE:-couchbase_only}
+    env_file:
+    - .env
```

Comment:
The reason we don't want to inject all the environment variables into each container is two fold:

1. Security. We want to ensure services only have access to the secrets they need
2. Future Planning. We will eventually break each of the listed services into a discrete service. At that time, we don't want to have an untracked reliance on a secret that could have been prevented by limited the access the service has to the secrets. With this type of information, generally, its better to be explicit than implicit. A scale solution would be using a vault or secrets manager but that's a future state.

This applied to all the other changes of the same kind in this PR.